lib/spree/responder.rb

```diff
@@ -10,7 +10,7 @@ def initialize(controller, resources, options = {})
       class_name = controller.class.name.to_sym
       action_name = options.delete(:action_name)
 
-      result = Spree::BaseController.spree_responders[class_name].
+      result = ApplicationController.spree_responders[class_name].
```

Comment:
There's a lot of bits of Spree code around "responders" that looks pretty nasty, and it's scattered all over the place. It would be great to assess it and see if we can tidy it up or remove it at some point. But not today... :+1: